src/components/DataTable/tests/DataTable.test.tsx

```diff
@@ -155,6 +155,15 @@ describe('<DataTable />', () => {
 
       totalsCells.forEach((total) => expect(total.text()).toBe(''));
     });
+
+    it('renders totals in the footer with a showTotalsInFooter prop', () => {
+      const totals = ['', '', ''];
+      const dataTable = mountWithAppProvider(
+        <DataTable {...defaultProps} totals={totals} showTotalsInFooter />,
+      );
+
+      expect(dataTable.find('thead tr')).toHaveLength(1);
```

Comment:
This is going to be true whether or not you pass `showTotalsInFooter` prop. Take a look at the way the markup is output and you'll see that this does not test the thing you want tested. If you make the `tfoot` conditionally rendered, you can change the test to:

```suggestion
      expect(dataTable.find('tfoot')).toHaveLength(1);
```

And it will accurately test the rendering you want.

Reply:
Good catch, I see the reasoning behind that now!

Reply:
@danrosenthal I think I've got it all set up now